build_artifacts/v1/v1.12/v1.12.2/cpu.pinned_env.in

```diff
@@ -0,0 +1,3 @@
+conda-forge::libmamba[version='>=1.5.11,<1.6.0']
```

Comment:
Is it possible to add comments to env.in files? If so can we add explanation for why these are pinned, to make it easier to determine whether we can unpin (e.g. libmamba causes build failures with 127 exit code). Or we can track this internally or via an Issue if that seems better.